uberfire-services/uberfire-services-backend/src/main/java/org/guvnor/common/services/backend/metadata/MetadataServiceImpl.java

```diff
@@ -67,12 +74,31 @@ public MetadataServiceImpl() {
     @Inject
     public MetadataServiceImpl(@Named("ioStrategy") IOService ioService,
                                @Named("configIO") IOService configIOService,
+                               CommentedOptionFactory commentedOptionFactory,
                                SessionInfo sessionInfo) {
         this.ioService = ioService;
         this.configIOService = configIOService;
+        this.commentedOptionFactory = commentedOptionFactory;
         this.sessionInfo = sessionInfo;
     }
 
+    @Override
+    public Path saveMetadata(final Path resource,
```

Comment:
In interface we use `final Path path` and here `final Path resource`. Could we please unify? To me second option sounds better, however feel free to pick any option.